sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -163,6 +163,9 @@ private[sql] class SessionState(sparkSession: SparkSession) {
   def executePlan(plan: LogicalPlan): QueryExecution = new QueryExecution(sparkSession, plan)
 
   def refreshTable(tableName: String): Unit = {
+    // Different from SparkSession.catalog.refreshTable, this API only refreshes the metadata.
+    // It does not reload the cached data. That means, if this table is cached as
+    // an InMemoryRelation, we do not refresh the cached data.
```

Comment:
`SharedState` can refresh the cached table data. In `SessionState`, we only can refresh the metadata. Thus, this API `refreshTable` only refresh the metadata